config/docker/nginx.conf

```diff
@@ -5,7 +5,7 @@ server {
     index index.html;
 
     # When requesting static paths with extension, try them, and return a 404 if not found
-    location ~ .+\.(css|js|html|png|jpg|jpeg|gif|bmp|ico|json|csv|otf|eot|svg|svgz|ttf|woff|woff2|ijmap|pdf|tif|map) {
+    location ~* .+\.(css|js|html|png|jpe?g|gif|bmp|ico|json|csv|otf|eot|svg|svgz|ttf|woff|woff2|ijmap|pdf|tif|map) {
```

Comment:
What does this extra `*` do? What's the difference between `location ~*` and `location ~`?

Reply:
Never mind. Just found the explanation here https://www.digitalocean.com/community/tutorials/understanding-nginx-server-and-location-block-selection-algorithms :sweat_smile: 

Good catch.